AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Edit.cshtml

```diff
@@ -99,7 +99,7 @@
                 <p><cite>Optional link to an outside page</cite></p>
             </div>
             <div class="col-md-10">
-                <input asp-for="ExternalUrl" class="form-control" />
+                <input type="url" asp-for="ExternalUrl" class="form-control" />

```

Comment:
I would prefer specifying the type using a data annotation on the ExternalUrl property of the view model. That way the input tag helper takes care of setting the type appropriately. 

In the ViewModel:

```
[Url]
property string ExternalUrl { get; set; }
```

More on the input tag helper here: http://www.davepaquette.com/archive/2015/05/13/mvc6-input-tag-helper-deep-dive.aspx